perception/radar_fusion_to_detected_object/schema/radar_fusion_to_detected_object.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Fusion to Detected Object Node",
+  "type": "object",
+  "definitions": {
+    "radar_fusion_to_detected_object": {
+      "type": "object",
+      "properties": {
+        "node_params": {
+          "type": "object",
+          "properties": {
+            "update_rate_hz": {
+              "type": "number",
+              "description": "INSERT_DESCRIPTION",
+              "default": 10.0,
+              "minimum": 0.0
+            }
+          },
+          "required": ["update_rate_hz"]
+        },
+        "core_params": {
+          "type": "object",
+          "properties": {
+            "bounding_box_margin": {
+              "type": "number",
+              "description": "The distance to extend the 2D bird's-eye view Bounding Box on each side. This distance is used as a threshold to find radar centroids falling inside the extended box. [m]",
+              "default": 2.0,
+              "minimum": 0.0
+            },
+            "split_threshold_velocity": {
+              "type": "number",
+              "description": "The object's velocity threshold to decide to split for two objects from radar information (currently not implemented) [m/s]",
+              "default": 5.0,
+              "minimum": 0.0
+            },
+            "threshold_yaw_diff": {
+              "type": "number",
+              "description": "The yaw orientation threshold. If $ \\vert \\theta {ob} - \\theta {ra} \\vert < threshold yaw_diff $ attached to radar information include estimated velocity, where $ \\theta{ob} $ is yaw angle from 3d detected object, $ \\theta_ {ra} $ is yaw angle from radar object. [rad]",
```

Comment:
Please reword to something that can be read in plain text format.

Reply:
Done.